src/nvim/eval.c

```diff
@@ -16258,6 +16260,13 @@ list_T *get_vim_var_list(int idx)
   return vimvars[idx].vv_list;
 }
 
+/// Get Dictionary v: variable value.  Caller must take care of reference count
+/// when needed.
+dict_T * get_vim_var_dict(int idx)
```

Comment:
There should not be a space after an asterisk. I would also suggest adding `FUNC_ATTR_PURE`.

I also do not see where this function is used. Are you sure it is needed at all?